{"abstract":[{"type":"text","text":"Called when the adapter failed to load the interstitial."}],"schemaVersion":{"patch":0,"minor":3,"major":0},"metadata":{"modules":[{"name":"CloudXCore"}],"externalID":"s:10CloudXCore27AdapterInterstitialDelegateP13didFailToLoad12interstitial5erroryAA0cD0_p_s5Error_ptF","fragments":[{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"text":"didFailToLoad","kind":"identifier"},{"text":"(","kind":"text"},{"text":"interstitial","kind":"externalParam"},{"kind":"text","text":": "},{"text":"AdapterInterstitial","kind":"typeIdentifier","preciseIdentifier":"s:10CloudXCore19AdapterInterstitialP"},{"text":", ","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"kind":"text","text":")"}],"roleHeading":"Instance Method","symbolKind":"method","title":"didFailToLoad(interstitial:error:)","required":true,"role":"symbol"},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"didFailToLoad"},{"text":"(","kind":"text"},{"text":"interstitial","kind":"externalParam"},{"kind":"text","text":": "},{"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitial","text":"AdapterInterstitial","preciseIdentifier":"s:10CloudXCore19AdapterInterstitialP","kind":"typeIdentifier"},{"kind":"text","text":", "},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"},{"parameters":[{"name":"interstitial","content":[{"inlineContent":[{"type":"text","text":"The interstitial that failed to load"}],"type":"paragraph"}]},{"name":"error","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"The error that caused the failure"}]}]}],"kind":"parameters"}],"variants":[{"paths":["\/documentation\/cloudxcore\/adapterinterstitialdelegate\/didfailtoload(interstitial:error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitialDelegate\/didFailToLoad(interstitial:error:)"},"sections":[],"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitialDelegate"]]},"kind":"symbol","references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"abstract":[],"type":"topic","role":"collection","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","title":"CloudXCore","url":"\/documentation\/cloudxcore","kind":"symbol"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/AdapterInterstitial":{"navigatorTitle":[{"kind":"identifier","text":"AdapterInterstitial"}],"type":"topic","role":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitial","kind":"symbol","title":"AdapterInterstitial","url":"\/documentation\/cloudxcore\/adapterinterstitial","abstract":[{"text":"Protocol for interstitial adapters. Interstitial adapters are responsible for loading and showing interstitial ads.","type":"text"}],"fragments":[{"text":"protocol","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AdapterInterstitial"}]},"doc://com.cloudx.sdk.core/documentation/CloudXCore/AdapterInterstitialDelegate":{"fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"AdapterInterstitialDelegate"}],"url":"\/documentation\/cloudxcore\/adapterinterstitialdelegate","abstract":[{"text":"Delegate for the interstitial adapter.","type":"text"}],"title":"AdapterInterstitialDelegate","navigatorTitle":[{"text":"AdapterInterstitialDelegate","kind":"identifier"}],"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitialDelegate","kind":"symbol","role":"symbol","type":"topic"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/AdapterInterstitialDelegate/didFailToLoad(interstitial:error:)":{"type":"topic","url":"\/documentation\/cloudxcore\/adapterinterstitialdelegate\/didfailtoload(interstitial:error:)","kind":"symbol","abstract":[{"text":"Called when the adapter failed to load the interstitial.","type":"text"}],"fragments":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"text":"didFailToLoad","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"interstitial"},{"text":": ","kind":"text"},{"text":"AdapterInterstitial","kind":"typeIdentifier","preciseIdentifier":"s:10CloudXCore19AdapterInterstitialP"},{"kind":"text","text":", "},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"title":"didFailToLoad(interstitial:error:)","required":true,"role":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/AdapterInterstitialDelegate\/didFailToLoad(interstitial:error:)"}}}